RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/utils/wxrc/wxrc.cpp,v
retrieving revision 1.24
diff -u -r1.24 wxrc.cpp
--- wxrc.cpp	2003/03/16 20:26:19	1.24
+++ wxrc.cpp	2003/07/17 17:53:26
@@ -169,7 +169,18 @@
         parFuncname = _T("InitXmlResource");
 
     for (size_t i = 0; i < cmdline.GetParamCount(); i++)
+    {
+#ifdef __WINDOWS__
+        wxString fn=wxFindFirstFile(cmdline.GetParam(i), wxFILE);
+        while (!fn.IsEmpty())
+        {
+            parFiles.Add(fn);
+            fn=wxFindNextFile();
+        }
+#else
         parFiles.Add(cmdline.GetParam(i));
+#endif
+    }
 }

 	  	 
